csharp-intermediate-topics/ConstVsReadonlyInCSharp/Tests/CircleCalculatorUnitTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using ConstVsReadonlyInCSharp;
+
+namespace Tests
+{
+    public class CircleCalculatorUnitTests
+    {
+        private CircleCalculator _circleCalculator = new CircleCalculator();
+
+        [Fact]
+        public void WhenCalculatingCircumferenceFor3_ThenReturn18_84()
+        {
+            var result = _circleCalculator.GetCircumference(3);
+
+            Assert.Equal(18.84, result);
```

Comment:
When dealing with "Equal" and floating point types, it is better to compare using a tolerance. Especially with .NET now being cross platform, depending on the processor where this code is running, it would be possible for there to be a slight variation within the result. This is a good discussion on the topic:
https://roundwide.com/equality-comparison-of-floating-point-numbers-in-csharp/